src/wallet/wallet.h

```diff
@@ -54,6 +54,9 @@ extern bool bSpendZeroConfChange;
 extern bool fSendFreeTransactions;
 extern bool fPayAtLeastCustomFee;
 extern unsigned int nAnchorConfirmations;
+// The maximum number of Orchard actions permitted within a single transaction.
+// This can be overridden with the -orchardactionlimit CLI flag.
```

Comment:
I don't think it's a CLI flag; it's a configuration option (command line or `zcash.conf`).